chaoslib/litmus/pod_delete/pod_failure_by_litmus.yml

```diff
@@ -33,15 +42,41 @@
 # If it is terminate means chaos is ended.
 - name: "[Prepare]: Setting the chaos_status to active"
   set_fact:
-    chaos_status: "active"
+    chaos_status: "active" 
+
+- block: 
+
+  - name: "[Prepare]: Including the litmus LIB for random pod deletion"
+    template:
+      src:  /chaoslib/litmus/pod_delete/pod-delete.j2
+      dest: /tmp/pod-delete.yml
+
+  - name: "[Prepare]: Creating timestamp pod for pod delete chaos"
+    shell: kubectl create -f /tmp/pod-delete.yml -n {{ c_ns }}
+    args:
+      executable: /bin/bash
+    register: helper_pod
+
+  - name: "[Status]: Checking whether application pods are in completed state"
+    shell: kubectl get pods -n {{ c_ns }} -l name=pod-delete-{{ run_id }} -o custom-columns=:.status.phase --no-headers
+    register: result
+    ignore_errors: yes
```

Comment:
can we use the chaos time diff to verify success or failure of the job? Refer the container kill exp. cc: @ispeakc0de